app/transformers/rural-payments-portal/applications-cs.js

```diff
@@ -0,0 +1,34 @@
+// CS stands for Countryside Stewardship
+export function transformOrganisationCSApplicationToBusinessApplications (applications) {
+  if (!applications) {
+    return { applications: [] }
+  }
+
+  if (applications.length === 0) {
+    return { applications: [] }
+  }
+
+  const result = []
+  for (const response in applications) {
+    result.push({
+      applicationStatus: {
+        id: response.application_id,
+        open: response.application_code,
```

Comment:
I think we will need to iron out whether this is the right mapping but for now it is fine